dataland-frontend/tests/component/components/forms/parts/elements/basic/UploadDocumentsForm.cy.ts

```diff
@@ -1,8 +1,10 @@
 import UploadDocumentsForm from "@/components/forms/parts/elements/basic/UploadDocumentsForm.vue";
-import { uploadDocuments } from "@sharedUtils/components/UploadDocuments";
```

Comment:
Why is it now UploadDocuments and no longer uploadDocuments?

Reply:
I made it a class instead of an object because due to multiple base data points in a form a single object to manage access to all UploadDocumentForms made not much sense to me. Now it's one object per UploadDocumentsForm.

Reply:
ok